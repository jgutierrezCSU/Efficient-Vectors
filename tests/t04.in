countOccLarge
10 12
3498533	3498534	3498533	3498536	3498537	3498531	3498532	3498527	3498533	3498532
countOccLarge
50 1000
2147450913	2147450913	2147450649	2147450649	2147450314	2147450649	2147450913	2147450314	2147450913	2147450649	2147450649	2147450649	2147450649	2147450649	2147450649	2147450314	2147450649	2147450913	2147450913	2147450649	2147450913	2147450314	2147450913	2147450314	2147450649	2147450649	2147450649	2147450314	2147450314	2147450913	2147450649	2147450649	2147450913	2147450314	2147450649	2147450649	2147450913	2147450649	2147450649	2147450314	2147450314	2147450649	2147450649	2147450913	2147450913	2147450314	2147450649	2147450649	2147450649	2147450913

