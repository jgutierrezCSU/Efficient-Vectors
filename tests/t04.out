Results of count occurrences of large integers:
3498537 1
3498536 1
3498534 1
3498533 3
3498532 2
3498531 1
3498527 1
Results of count occurrences of large integers:
2147450913 14
2147450649 25
2147450314 11
